src/infra/dex/balancer/mod.rs

```diff
@@ -1,16 +1,26 @@
 use {
     crate::{
-        domain::{auction, dex, eth, order},
+        domain::{
+            auction,
+            dex,
+            eth::{self, TokenAddress},
+            order,
+        },
         util,
     },
     contracts::ethcontract::I256,
     ethereum_types::U256,
     ethrpc::current_block::CurrentBlockStream,
-    std::sync::atomic::{self, AtomicU64},
+    num::ToPrimitive,
+    std::{
+        ops::Add,
+        sync::atomic::{self, AtomicU64},
+        time::Duration,
+    },
     tracing::Instrument,
 };
 
-mod dto;
+pub mod dto;
```

Comment:
I think this is only `pub` now to reuse the `QUERY`, right?
This could lead to us changing the query unintentionally without causing any tests to fail. Maybe we should leave the module private and simply copy over the query into the test module. That way if we get these out of sync at least we get notified about it and have to make it explicit that the new version is indeed correct.

Reply:
I made the QUERY private and copied it to the tests. But dto still needs to be public because HumanReadableAmount is also used in tests.

Reply:
True but there is no reason this has to be like this, right?
I mean we could also just configure 2 strings instead of using the human readable type, no?

Reply:
Made it private.